FE/src/pages/Search/components/SearchAgentCard.tsx

```
@@ -50,18 +63,20 @@ const SearchAgentCard = ({
-          <span className="title-m-bold -mt-2">{agent.name} 행정사</span>
+          <span className="title-m-bold -mt-2">{agent.agentName} 행정사</span>
           <div className="flex-col flex gap-1.5">
             <span className="flex flex-row items-center gap-1.5 caption-m-medium">
               <IcGraduation size={14} /> 전문 분야
             </span>
             {authed ? (
               <ol className="flex flex-row gap-1">
-                {agent.jobs.slice(0, 2).map((jobId) => (
+                {agent.agentSpecialityTop2.slice(0, 2).map((jobId) => (
                   <Tag type={"small_fill_violet_max"}>{jobList[jobId]}</Tag>
                 ))}
```

Biome (2.3.13)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)